android/build.gradle

```diff
@@ -53,6 +53,9 @@ repositories {
   jcenter()
   google()
 
+  // @rob do we need this?
+  maven { url "https://jitpack.io" }
```

Comment:
@rbscott do we need this?

Reply:
@ccorcos  Yeah, that was how I found the dependency for Persistent Cookie.